lib/rollbax/item.ex

```diff
@@ -4,10 +4,15 @@ defmodule Rollbax.Item do
     %{"access_token" => token,
       "data" => %{
         "server" => %{
-          "host" => List.to_string(host)},
+          "host" => List.to_string(host)
+        },
         "environment" => envt,
-        "language"  => "Elixir",
-        "framework" => "OTP"}}
+        "language"  => language(),
+        "framework" => framework(),
```

Comment:
I really not sure that we should put Erlang/OTP version here (as `framework`), it is rather a platform (there is Erjang available for example).
But then we need to invent another naming for OS information or maybe wait a bit before someone will be needing it. Thoughts?
\cc @whatyouhide